content/en/tracing/troubleshooting/_index.md

```diff
@@ -124,6 +124,34 @@ kubectl exec -it <agent-pod-name> -c trace-agent -- agent flare <case-id> --loca
 
     Knowing what versions are being used allows us to ensure integrations are supported in our [Compatiblity Requirements][15] section, check for known issues, or to recommend a tracer or language version upgrade if it will address the problem.
 
+## Service naming convention issues
+
+If you run into quota-related issues, try following the steps below to ensure you're following best practices for service naming conventions.
+
+### Exclude environment tag values from service names
+
+By default, the environment (`env`) is the primary tag for Datadog APM.
+
+{{< img src="/tracing/troubleshooting/troubleshooting-service-naming-convention-issues-3.png" alt="Environment is the default primary tag" style="width:100%;" >}}
+
+A service is typically deployed in multiple environments, such as `prod`, `staging`, and `dev`. Performance metrics like request counts, latency, and error rate differ across various environments. The environment dropdown in the Service Catalog allows you to scope the data in the **Performance** tab to a specific environment.
+
+{{< img src="/tracing/troubleshooting/troubleshooting-service-naming-convention-issues-2.png" alt="Choose a specific environment using the `env` dropdown in the Service Catalog" style="width:100%;" >}}
+
+One pattern that often leads to issues with an overwhelming number of services is to include the environment value in service names. For example, you might have two unique services instead of one since they are operating in two separate environments: `prod-web-store` and `dev-web-store`.
+
+You should avoid this pattern and rename your services by tuning your instrumentation.
+
+**Note**: Datadog's trace metrics are unsampled, which means the app shows all data instead of subsections of them. The [volume guidelines][17] are applied to ensure that we consistently provide users with an optimal experience across the app.
```

Comment:
The second sentence in this note sounded internal facing to me, feel free to reword.

```suggestion
Trace metrics are unsampled, which means your instrumented application shows all data instead of subsections of them. The [volume guidelines](#data-volume-guidelines) are also applied. 
```